src/Core/DevOps/System/Command/SystemDumpDatabaseCommand.php

```diff
@@ -30,16 +38,26 @@ public function execute(InputInterface $input, OutputInterface $output): int
         $params = $this->connection->getParams();
 
         $path = sprintf('%s/%s_%s.sql', $this->defaultDirectory, $params['host'], $dbName);
+        $ignoredTables = [];
+
+        foreach ((array) $input->getOption('ignore-table') as $ignorableTable) {
```

Comment:
Please outsource this into a new function to initialize the result directly into $ignoredTables :) Then you could return early an empty array, when the option 'ignore-table' is not set.

Reply:
I extracted and rewrote it. Please re-review :)